packages/modeling/src/operations/extrusions/extrudeLinearGeom2.js

```diff
@@ -7,7 +7,7 @@ const slice = require('./slice')
 
 const extrudeFromSlices = require('./extrudeFromSlices')
 
-/*
+/**
```

Comment:
This function is not part of the extrusions (index.js), so should probably be left out of user documentation.

Also, due to the object orientated slant of JSDOC, we have to provide an alias for each function in order for the function to have a proper name space.

`@alias module:modeling/extrusions.extrudeLinear`

Reply:
@z3dev my intention was not to include this in user documentation. I thought I was just fixing typos. I will undo the comment changes in this PR.

Its a bummer if we can't use jsdoc `/**` comments for private functions though. Without the double-asterisk, many IDEs won't show autocomplete hints or syntax highlighting, and many linters won't pickup bugs.

With `/*` non-jsdoc comment (no autocomplete hint, no jsdoc coloring):
![no-jsdoc](https://user-images.githubusercontent.com/1766297/155255497-1acfee3d-b625-4d61-86aa-89ca5843eeee.png)

Wtih `/**` jsdoc comment (full autocomplete, jsdoc color hints):
![jsdoc](https://user-images.githubusercontent.com/1766297/155255505-ab3c26a8-f212-4585-8174-f26419e20d07.png)

Reply:
That is a bummer. IDE highlights are nice hints for developers.

Reply:
I found @ignore in the JSDOC manual. This might allow the IDE to use the docs, while removing the function from documentation.

https://jsdoc.app/tags-ignore.html